### PR TITLE
OCPQE-27376 Add OCP 4.18 rollback jobs

### PR DESCRIPTION
**Pull request by JianLi-RH**
https://issues.redhat.com/browse/OCPQE-27376

We need to add 4.18 z stream rollbacks, including nightly and stable rollback, the initial version is always the oldest stable version: https://github.com/openshift/release/pull/55692#discussion_r1732036820

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-nightly-azure-upi-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-stable-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-nightly-ibmcloud-ipi-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-stable-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-nightly-azure-upi-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-stable-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-nightly-ibmcloud-ipi-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-stable-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-nightly-gcp-ipi-ovn-ipsec-f7

**Comment by JianLi-RH** [comment]
4.17 stable rollback passed: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/59429/rehearse-59429-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-stable-aws-ipi-byo-route53-f7/1863834336491474944

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-nightly-azure-upi-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-nightly-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-stable-aws-ipi-byo-route53-f7

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-rollback-nightly-ibmcloud-ipi-f7

**Comment by JianLi-RH** [comment]
rollback is triggered in 4.18 nightly job but there is olm issue: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/59429/rehearse-59429-periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-rollback-nightly-aws-ipi-byo-route53-f7/1864119562920792064

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      version: "4.18"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 1Gi
+tests:
+- as: aws-ipi-byo-route53-f7
```
Comment (jiajliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-nightly.yaml:62]: do we still need the test frequency to be f7? cc @shellyyang1989 to confirm

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-nightly.yaml:62]: It's relatively stable now, so we can lower the frequency as needed.

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-nightly.yaml:62]: shall we use `f28` for all existing rollback jobs, including 4.16, 4.17 and 4.18.

Comment (jiajliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-rollback-nightly.yaml:62]: I'm ok with `f28`

**Pull request merged**